tests/pr_test.go

```diff
@@ -167,7 +77,7 @@ func TestRunUpgradeFullyConfigurable(t *testing.T) {
 	t.Parallel()
 
 	var region = validRegions[rand.Intn(len(validRegions))]
-	prefix := fmt.Sprintf("icm-da-up-%s", strings.ToLower(random.UniqueId()))
+	prefix := "icm-da-up-"
```

Comment:
please change this back and then update line 140 to use the `prefix` (instead of `options.Prefix`)